README.md

```diff
@@ -69,35 +69,39 @@ $ git clone https://github.com/nvim-treesitter/nvim-treesitter.git
 
 ## Adding parsers
 
-Treesitter uses a different _parser_ for every language. It can be quite a pain to install, but fortunately `nvim-treesitter`
-provides two command to tackle this issue:
+Treesitter uses a different _parser_ for every language, which needs to be generated via `tree-sitter-cli` from a `grammar.js` file, then compiled to a `.so` library that needs to be placed in neovim's `runtimepath` (typically under `parser/{lang}.so`). To simplify this, `nvim-treesitter`
+provides commands to automate this process:
 
-- `TSInstall {language}` to install one or more parsers.
-  `TSInstall <tab>` will give you a list of supported languages, or select `all` to install them all.
-- `TSInstallInfo` to know which parser is installed.
-- `TSUpdate` to update already installed parsers
+- `TSInstallInfo` to know which parsers are available and installed.
+- `TSInstall {language}` to install one or more parsers from a generated `c` file. (This requires a `C` compiler in your path.)
```

Comment:
> from a generated `c` file

I guess it's a bit unclear to what file this refers. It sounds a bit as if the user should provide that file. 

> to install one or more parsers from the project's Github repository

Do we have to repeat the compiler requirement that we mentioned in the first README section?

Reply:
True (although that section doesn't state the requirement for `TSInstallFromGrammar` -- so maybe that section should be "minimal requirements" instead?)

Agreed on the unclear wording; I'll improve it.